core/com.b2international.snowowl.datastore/src/com/b2international/snowowl/core/repository/RepositoryPlugin.java

```diff
@@ -221,6 +230,20 @@ public void preRun(SnowOwlConfiguration configuration, Environment env) {
 		
 	}
 	
+	private void registerRequestMetrics(MeterRegistry registry, IEventBus eventBus) {
+		FunctionCounter.builder("requests.completed", eventBus, bus -> bus.getFinishedMessages(Request.TAG))
+				.description("The approximate total number of requests that have completed execution")
```

Comment:
Is this an approximate metric or exact? Looks like exact to me. Same for the others.